models/channel/src/channel-draft/presets/sample-data-fashion/store-other.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { TChannelDraft, TChannelDraftGraphql } from '../../../types';
+import storeOther from './store-other';
+
+describe('store-other channel', () => {
+  it('should match REST snapshot', () => {
+    const channel = storeOther().build<TChannelDraft>();
+
+    expect(channel).toMatchInlineSnapshot(`
+      {
+        "address": undefined,
+        "custom": undefined,
```

Comment:
I wonder if we'll need some sort of prefiltering to clear these undefined fields when we make a query. Right now we sort of generate everything.

Reply:
After checking out other preset tests, they don't use `toMatchInlineSnapshot`, which means you don't have to bother checking fields that aren't set.